.github/workflows/flowzone.yml

```
@@ -0,0 +1,45 @@
+name: Flowzone
+
+on:
+  pull_request:
+    types: [opened, synchronize, closed]
+    branches:
+      - "main"
+      - "master"
+
+jobs:
+  flowzone-staging:
+    name: balena-staging
+    uses: product-os/flowzone/.github/workflows/flowzone.yml@protect
```

This needs to go back to
```suggestion
    uses: product-os/flowzone/.github/workflows/flowzone.yml@master
```
once working and https://github.com/product-os/flowzone/pull/208 is merged